typescript/sdk/src/hook/types.ts

```diff
@@ -3,12 +3,14 @@ import { BigNumber } from 'ethers';
 import { Address } from '@hyperlane-xyz/utils';
 
 import { IgpConfig } from '../gas/types';
+import { ChainName } from '../types';
 
 export enum HookType {
   MERKLE_TREE = 'merkleTreeHook',
   INTERCHAIN_GAS_PAYMASTER = 'interchainGasPaymaster',
   AGGREGATION = 'aggregationHook',
   PROTOCOL_FEE = 'protocolFee',
+  OP_STACK = 'opStack',
```

Comment:
we use this for the address serialization so let's add the hook suffix for clarity

Reply:
done